org.liri.liri.json

```diff
@@ -0,0 +1,56 @@
+{
+  "id": "org.liri.liri",
+  "runtime": "org.freedesktop.Platform",
+  "runtime-version": "23.08",
+  "sdk": "org.freedesktop.Sdk",
+  "command": "Li-ri",
+  "finish-args": [
+    "--socket=pulseaudio",
+    "--share=ipc",
+    "--socket=fallback-x11",
+    "--socket=wayland",
+    "--device=dri"
+  ],
+  "modules": [
+    "shared-modules/SDL2/SDL2-with-libdecor.json",
+    {
+      "name": "libmodplug",
+      "sources": [
+        {
+          "type": "archive",
+          "url": "https://sourceforge.net/projects/modplug-xmms/files/libmodplug/0.8.9.0/libmodplug-0.8.9.0.tar.gz",
+          "sha256": "457ca5a6c179656d66c01505c0d95fafaead4329b9dbaa0f997d00a3508ad9de"
+        }
+      ]
+    },
+    {
+      "name": "sdl_mixer",
+      "config-opts": [
+        "--enable-music-mod-modplug=yes"
+      ],
+      "sources": [
+        {
+          "type": "archive",
+          "url": "https://github.com/libsdl-org/SDL_mixer/releases/download/release-2.6.2/SDL2_mixer-2.6.2.tar.gz",
+          "sha256": "8cdea810366decba3c33d32b8071bccd1c309b2499a54946d92b48e6922aa371"
+        }
+      ]
+    },
```

Comment:
```suggestion
```

It's in 23.08 runtime https://gitlab.com/freedesktop-sdk/freedesktop-sdk/-/blob/05b660171170dbd3fd80d27ac578c36ca3e9eb21/elements/platform.bst#L157

Reply:
It seems the runtime does not compile/link with modplug and sounds are not played for my application if I remove it

Reply:
Hm you're right, in case keep it.